src/laminar.cpp

```diff
@@ -338,9 +340,13 @@ std::string Laminar::getStatus(MonitorScope scope) {
         j.set("description", desc == jobDescriptions.end() ? "" : desc->second);
     } else if(scope.type == MonitorScope::ALL) {
         j.startArray("jobs");
-        db->stmt("SELECT name,number,startedAt,completedAt,result,reason FROM builds b "
-                 "JOIN (SELECT name n,MAX(number) latest FROM builds WHERE result IS NOT NULL GROUP BY n) q "
-                 "ON b.name = q.n AND b.number = latest")
+        db->stmt("SELECT name"
+                 " , first_value(number) OVER (PARTITION BY name ORDER BY number DESC)"
+                 " , first_value(startedAt) OVER (PARTITION BY name ORDER BY number DESC)"
+                 " , first_value(completedAt) OVER (PARTITION BY name ORDER BY number DESC)"
+                 " , first_value(result) OVER (PARTITION BY name ORDER BY number DESC)"
+                 " , first_value(reason) OVER (PARTITION BY name ORDER BY number DESC)"
+                 " FROM builds WHERE result IS NOT NULL GROUP BY name")
```

Comment:
Suggest `WINDOW win_name AS (PARTITION BY name ORDER BY number DESC)` to avoid repetition

Reply:
Good idea, done.

Reply:
Actually, if the index is what makes the difference, then perhaps the clearest way to write this query would be more like `SELECT ... FROM builds GROUP BY name HAVING number = MAX(number)` (sorry can't experiment right now)

Reply:
And simplified the query with your suggestion. It runs as fast as the previous versions.